lib/rubocop/cop/cable_ready/application_record_enable_updates.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module CableReady
+      class ApplicationRecordEnableUpdates < Base
+        MSG = "enable_updates should not be switched on globally"
+
+        def_node_search :active_record_base_class?, <<~PATTERN
+          (const (const nil? :ActiveRecord) :Base)
+        PATTERN
+
+        def_node_matcher :is_application_record?, <<~PATTERN
+          (const nil? :ApplicationRecord)
+        PATTERN
+
+        def on_send(node)
+          receiver_node, _method_name, *_arg_nodes = *node.parent.parent
+
+          return if node.method_name != :enable_updates
```

Comment:
This changed with CableReady `5.0.0.pre10`:
```suggestion
          return if node.method_name != :enable_cable_ready_updates
```
or maybe we can check for both?